src/mpyl/build.py

```diff
@@ -138,7 +138,7 @@ def run_mpyl(
                 accumulator=run_result,
                 executor=steps,
                 reporter=reporter,
-                dry_run=cli_parameters.dryrun or cli_parameters.local,
+                install=cli_parameters.install,
```

Comment:
Shouldn't the `local` parameter (whatever that is) keep disabling the `install` flag?
```suggestion
                install=cli_parameters.install and not cli_parameters.local,
```

If I had to guess, I'd say the original idea is to never run `helm install` from your local machine.

Reply:
Well you could always bypass this already. `local` was based on `not ci` so if you passed `--ci` you could also run `helm install` locally. Which is kinda fine, maybe you want to test it on a local minikube cluster or something right? In this case `install` is `False` by default, so you specifically have to choose to pass `--install`. In which case I assume you know you'll trigger `helm install`.
Also, `--dryrun` worked the other way around, it was `False` by default so you could trigger `helm install` without purposely passing a flag. So in that old situation the `or` makes more sense, now the handholding is not needed as much.

Anyway, very long answer for a simple change, but I did think about this change before applying it 😅 haha

Reply:
That makes sense, thanks for explaining!